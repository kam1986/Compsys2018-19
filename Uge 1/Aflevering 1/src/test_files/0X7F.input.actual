test_files/0X7F.input: data
