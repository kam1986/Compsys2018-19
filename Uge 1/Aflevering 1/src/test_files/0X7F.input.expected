test_files/0X7F.input: International EBCDIC text, with no line terminators
